### ISSUE TITLE
compilation error with omake 0.10.1

### ISSUE THREAD

**Issue by Khady** [mfp/extprot#20]
```
$ omake 
*** omake: reading OMakefiles
*** omake error:
   File Omake_builtin: line 0, characters 0-0
   include file Pervasives not found in OMAKEPATH (OMAKEPATH[] = "." : Sequence /home/user/.opam/4.02.3/lib/omake : Dir)
```

I am sorry not to provide a patch, I don't know how to use omake.
It's compiling fine with omake.0.9.8.6-0.rc1

**Comment by mfp** [comment]
On Mon, Nov 21, 2016 at 10:16:34PM -0800, Louis Roché wrote:
> ```
> $ omake 
> *** omake: reading OMakefiles
> *** omake error:
>    File Omake_builtin: line 0, characters 0-0
>    include file Pervasives not found in OMAKEPATH (OMAKEPATH[] = "." : Sequence /home/user/.opam/4.02.3/lib/omake : Dir)
> ```

This looks like a broken OMake install at first sight (Pervasives is, like
OCaml's module of the same name, the very core of the standard lib). Will try
to reproduce with 0.10.1 and determine whether it's a blatantly
backwards-incompatible change (unlikely), a packaging issue or something else.

-- 
Mauricio Fernández

**Comment by rgrinberg** [comment]
This was an issue with a temporarily broken install of omake 0.10.1 and affected all packages. The currently released omake 0.10.1 fixes this.
